docs/howto/krustlet-on-microk8s.md

```diff
@@ -0,0 +1,141 @@
+# Running Krustlet on [microk8s](https://microk8s.io)
+
+These are steps for running krustlet node(s) and [microk8s](https://microk8s.io) on the same machine.
+
+## Prerequisites
+
+You will require a running microk8s cluster for this guide. The steps below assume you will run microk8s and the Krustlet, on a single machine. `kubectl` is required but is installed with microk8s as `microk8s.kubectl`. The following instructions use `microk8s.kubectl` for simplicity. You may use a standlone `kubectl` if you prefer.
+
+## Step 1: Create a service account user for the node
+
+We will need to create a Kubernetes configuration file (known as the kubeconfig) and service account
+for krustlet to use to register nodes and access specific secrets.
+
+```shell
+SERVICE_ACCOUNT_NAME="krustlet"
+CONTEXT="krustlet"
+CLUSTER="microk8s-cluster"
+NAMESPACE="kube-system"
+USER="${CONTEXT}-token-user"
+
+SECRET_NAME=$(microk8s.kubectl get serviceaccount/${SERVICE_ACCOUNT_NAME} \
+  --namespace=${NAMESPACE} \
+  --output=jsonpath='{.secrets[0].name}')
+TOKEN_DATA=$(microk8s.kubectl get secret/${SECRET_NAME} \
+  --namespace=${NAMESPACE} \
+  --output=jsonpath='{.data.token}')
+
+TOKEN=$(echo ${TOKEN_DATA} | base64 -d)
+
+# Create user
+microk8s.kubectl config set-credentials ${USER} \
+--token ${TOKEN}
+# Create context
+microk8s.kubectl config set-context ${CONTEXT}
+# Set context to use cluster, namespace and user
+microk8s.kubectl config set-context ${CONTEXT} \
+--cluster=${CLUSTER} \
+--user=${USER} \
+--namespace=${NAMESPACE}
+```
+
+> **NB** We'll switch to this context when we run krustlet; for now we'll continue using the current (probably 'default') context
```

Comment:
Should this just be `**NOTE**`?

Reply:
Fixed 7faf57a47d21ad80ad040384bb936fa1c9789d44